src/util/BaseLoginController.js

```diff
@@ -92,8 +92,14 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
     toggleButtonState: function (state) {
       var button = this.$el.find('.button');
       button.toggleClass('link-button-disabled', state);
-    }
+    },
 
+    // Event triggered at after a page is rendered along with the classname to identify the page
+    postRender: function() {
+      _.defer(_.bind(function () {
+        this.trigger('pageRenderedEvent',[this.className]);
```

Comment:
Double checking - does every controller set it's className correctly?

Also, spacing.